test/ui-automation/test/specs/oidc-flow.js

```diff
@@ -96,9 +96,7 @@ describe("TrustBloc Wallet - OIDC Share flow", async function () {
     const shareBtn = await $("#share-credentials");
     await shareBtn.waitForExist();
     await shareBtn.click();
-
-    // TODO validate success message
-    const msg = await $("b*=ERROR: failed to validate presentation : JSON unmarshalling of verifiable presentation: unexpected end of JSON input");
+    const msg = await $("#success-share-message");
```

Comment:
The html is set by Golang template (https://github.com/trustbloc/wallet/pull/1620/files#diff-7776269460643279387337ba77e8928e056c71dc016d07741d39bea37dc106ceL6). In case of failure, does the the tag/id present in html with empty message or tag/id doesn't exist?

If it exists for all the cases, then it would be better to validate the message rather than id in tests.